src/iter/traverser.rs

```diff
@@ -26,11 +26,11 @@ pub(crate) struct NodeIter<'g, K, V> {
     /// Initial table size
     base_size: usize,
 
-    guard: &'g Guard,
+    guard: &'g Guard<'g>,
 }
 
 impl<'g, K, V> NodeIter<'g, K, V> {
-    pub(crate) fn new(table: Shared<'g, Table<K, V>>, guard: &'g Guard) -> Self {
+    pub(crate) fn new(table: Shared<'g, Table<K, V>>, guard: &'g Guard<'_>) -> Self {
```

Comment:
Should this not be `'g` instead of `'_`?

Reply:
I don't think so? We only care about the lifetime of the guard object, not the lifetime of the reference to the collector it holds.